crates/rolldown_common/src/inner_bundler_options/types/optimization.rs

```diff
@@ -103,7 +103,7 @@ where
   let deserialized = Option::<Value>::deserialize(deserializer)?;
   match deserialized {
     Some(Value::Bool(v)) => Ok(Some(InlineConstOption::Bool(v))),
-    Some(Value::String(s)) if s == "safe" => Ok(Some(InlineConstOption::Safe)),
+    Some(Value::String(s)) if s == "smart" => Ok(Some(InlineConstOption::Safe)),
```

Comment:
> `InlineConstOption::Safe`

Should we rename the internal ones as well?

Reply:
You are right.